spec/jobs/resolution_proofing_job_spec.rb

```diff
@@ -529,11 +529,11 @@
         expect(result[:success]).to be true
         expect(result[:exception]).to be_nil
         expect(result[:timed_out]).to be false
-        expect(result[:threatmetrix_review_status]).to eq('pass')
+        expect(result[:threatmetrix_review_status]).to eq('fail')
```

Comment:
I just double-checked and this should `reject`, not `fail` (technically the valid values for `review_status` are `pass`, `review`, `reject`).

Reply:
Changed. I'm assuming the missing pii case should 'reject' as well.